hbp_validation_framework/utils.py

```diff
@@ -70,11 +82,295 @@ def _make_js_file(data):
         json.dump(data, outfile)
         outfile.write("'")
 
-def run_test(hbp_username="", hbp_password=None, environment="production", model="", test_instance_id="", test_id="", test_alias="", test_version="", storage_collab_id="", register_result=True, client_obj=None, **test_kwargs):
+def prep_run_test(hbp_username="", hbp_password=None, environment="production", test_instance_id="", test_id="", test_alias="", test_version="", client_obj=None, **params):
```

Comment:
For consistency with `__init__.py`, use `username` and `password`

Reply:
Done